src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -273,7 +277,7 @@ static int _PaPulseAudio_ProcessAudio(PaPulseAudio_Stream *stream,
 
     if( stream->outputStream )
     {
-        /* Calculate how many bytes goes to one frame */
+        /* Calculate how many bytes go into to one frame */
```

Comment:
This comment is not consistent with the code. I think that the code is calculating the size of the host buffer in bytes.
```suggestion
        /* Calculate the host buffer size in bytes */
```